week-1/H-numbers/exercise.js

```diff
@@ -1 +1,4 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents=26;
+var numberOfMentors=15;
+console.log(numberOfStudents+numberOfMentors);
```

Comment:
Thast prints the total number, yes, great!
But the instructions say that this should be the output:
```Number of students: 15
Number of mentors: 8
Total number of students and mentors: 23```

So how can you fix it?